release_notes.md

```diff
@@ -2,15 +2,11 @@
 <!-- Please add your release notes in the following format:
 - My change description (#PR)
 -->
-- Update Python Worker Version to [4.7.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.7.0)
-- Update Python Worker Version to [4.6.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.6.0)
-- Update Python Worker Version to [4.5.0](https://github.com/Azure/azure-functions-python-worker/releases/tag/4.5.0)
+
 - Update Node.js Worker Version to [3.5.0](https://github.com/Azure/azure-functions-nodejs-worker/releases/tag/v3.5.0)
-- Host support for out-of-proc cancellation tokens ([#2153](https://github.com/Azure/azure-functions-host/issues/2152))
-- Updated Java Worker Version to [2.4.0](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.4.0)
 - Update PowerShell Worker 7.2 to 4.0.2258 [Release Note](https://github.com/Azure/azure-functions-powershell-worker/releases/tag/v4.0.2258)
```

Comment:
Did the changes go out with 4.11? Trying to understand why the file isn't reset here.

Reply:
I am updating it to what is in `dev` but not in `release` - this is because I didn't reset the release notes immediately after the last release.
https://github.com/Azure/azure-functions-host/compare/release/4.x...dev